invoice.go

```
@@ -355,11 +365,99 @@ func (r *GetCustomerInvoiceV1Request) Do() (GetCustomerInvoiceV1Response, error)
 
+// DoAll performs the request across all result pages and returns every invoice combined into a
+// single response, in page order. It always fetches at the API's default page size. An initial
+// "find" request retrieves page 1 to read the total count and page size from the metadata; the
+// remaining pages are then fetched by a fixed pool of at most Client.Concurrency workers (default
+// DefaultConcurrency), so the concurrency is bounded and independent of the number of pages. Any
+// caller-set query parameters (documentType, status, ...) are preserved on every page; a
+// caller-set page size is ignored.
+func (r *GetCustomerInvoiceV1Request) DoAll() (GetCustomerInvoiceV1Response, error) {
+	var base GetCustomerInvoiceV1QueryParams
+	if r.queryParams != nil {
+		base = r.queryParams.(GetCustomerInvoiceV1QueryParams) // panics if queryParams is non-nil and wrong type
```

golangci-lint (2.12.2)

[error] type assertion must be checked

(forcetypeassert)